config/locales/find.yml

```diff
@@ -113,6 +113,10 @@ en:
           above_or_equivalent_qualification: or above, or equivalent qualification
       about_course:
         heading: Course details
+      qualifications_summary_component:
+        view:
+          tda_with_qts: Teacher degree apprenticeship (TDA) with QTS
+          tda_with_qts_content_html: On a teacher degree apprenticeship you’ll work in a school and earn a salary while getting a bachelor’s degree and qualified teacher status (QTS). <a class="govuk-link" href="https://getintoteaching.education.gov.uk/train-to-be-a-teacher/teacher-degree-apprenticeships">Find out more about teacher degree apprenticeships</a>
```

Comment:
It would be nice to maintain our pattern of keeping the GIT links here so that it's easier to audit/change later on: 

https://github.com/DFE-Digital/publish-teacher-training/blob/06a279cc245d8660dff1dadc5d941ed77fea416e/config/locales/find.yml#L236